docs/v3/source/includes/resources/apps/_clear_buildpack_cache.md.erb

```diff
@@ -0,0 +1,36 @@
+### Clear buildpack cache for application
+
+```
+Example Request
+```
+
+```shell
+curl "https://api.example.org/v3/apps/[guid]/actions/clear_buildpack_cache" \
+  -X POST \
+  -H "Authorization: bearer [token]"
+```
+
+```
+Example Response
+```
+
+```http
+HTTP/1.1 202 Accepted
+Location: https://api.example.org/v3/jobs/[guid]
+```
+
+This endpoint will delete all of the existing buildpack caches for a specified app.
+The buildpack cache is used during staging by buildpacks as a way to
+cache certain resources, e.g. downloaded Ruby gems. A user who wants to
+decrease the size of their blobstore could use this endpoint to delete
+unnecessary blobs.
```

Comment:
The last sentence makes sense only for the global cache deletion.
The main reason for deleting a buildpack cache is that an app doesn't stage anymore due to out-of-disk caused by a large buildpack cache content. Often the real RC is that the buildpack didn't clean up e.g. old runtime versions that are not used/needed anymore.